cmd/unused/internal/ui/group_table.go

```diff
@@ -0,0 +1,64 @@
+package ui
+
+import (
+	"context"
+	"fmt"
+	"os"
+	"strings"
+	"text/tabwriter"
+
+	"github.com/grafana/unused"
+)
+
+func GroupTable(ctx context.Context, options Options) error {
+	disks, err := listUnusedDisks(ctx, options.Providers)
+	if err != nil {
+		return err
+	}
+
+	if options.Filter.Key != "" {
+		filtered := make(unused.Disks, 0, len(disks))
+		for _, d := range disks {
+			if d.Meta().Matches(options.Filter.Key, options.Filter.Value) {
+				filtered = append(filtered, d)
+			}
+		}
+		disks = filtered
+	}
+
+	if len(disks) == 0 {
+		fmt.Println("No disks found")
+		return nil
+	}
+
+	w := tabwriter.NewWriter(os.Stdout, 8, 4, 2, ' ', 0)
+
+	headers := []string{"PROVIDER", "GROUP_BY", "TYPE", "SIZE_GB"}
```

Comment:
I'm not sure `GROUP_BY` is clear enough as a column header. I'd love to have the value passed by the user for the `-group-by` flag, although I'm not sure how would that look in the case of something like `kubernetes.io/created-for/pvc/namespace` for instance.

Reply:
I have doubts here as well and can't find good enough solution for column name.
Tag names are long and also confusing when I tried to use them.

I have 3 ideas:
- add comment about grouping tag when run with `-v` flag and use `GROUP_BY_TAG` as column name
  ```
  Grouping by 'kubernetes.io/created-for/pvc/namespace' tag.
  PROVIDER  GROUP_BY_TAG    TYPE    DISKS_COUNT  TOTAL_SIZE_GB
  GCP       redis-test      ssd     3            9
  ```
- print group-by name as column
  ```
  PROVIDER  kubernetes.io/created-for/pvc/namespace  TYPE    DISKS_COUNT  TOTAL_SIZE_GB
  GCP       redis-test                               ssd     3            9
  ```
- print upcased group-by name
  ```
  PROVIDER  KUBERNETES.IO/CREATED-FOR/PVC/NAMESPACE  TYPE    DISKS_COUNT  TOTAL_SIZE_GB
  GCP       redis-test                               ssd     3            9
  ```

what do you think?